common/app/views/fragments/containers/facia_cards/standardHeaderMeta.scala.html

```diff
@@ -29,7 +29,7 @@
                                 </div>
                             }
                             @if(isAUSElection) {
-                                <div class="badge-slot badge-slot--us-election">
+                                <div class="badge-slot badge-slot--aus-election">
```

Comment:
Do we need a separate elements for this? Couldn't we just have a generic `badge-slot` and change the image inside it?